ddp-workspace/projects/ddp-atcp/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,103 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+
+import {
+  Auth0CodeCallbackComponent,
+  AuthGuard,
+  IrbGuard,
+  BrowserGuard
+} from 'ddp-sdk';
+
+import { LoginLandingRedesignedComponent } from 'toolkit';
+import * as RouterResource from './router-resources';
+
+import { WelcomeComponent } from './components/welcome/welcome';
+import { AboutUsComponent } from './components/about-us/about-us';
+import { JoinUsComponent } from './components/join-us/join-us';
+import { AboutInitiativeComponent } from './components/about-initiative/about-initiative';
+import { DataAccessComponent } from './components/data-access/data-access';
+import { PasswordComponent } from './components/password/password';
+import { DashBoardComponent } from './components/dashboard/dashboard';
+import { ErrorComponent } from './components/error/error';
+import { StatisticsComponent } from './components/statistics/statistics';
+
+const routes: Routes = [
+  {
+      path: RouterResource.Error,
+      component: ErrorComponent,
+      canActivate: [IrbGuard]
+  },
+  {
+      path: RouterResource.Statistics,
+      component: StatisticsComponent,
+      canActivate: [
+        IrbGuard,
+        BrowserGuard,
+        AuthGuard
+      ]
+  },
+  {
+    path: RouterResource.Dashboard,
+    component: DashBoardComponent,
+    canActivate: [
+      IrbGuard,
+      BrowserGuard,
+      AuthGuard
+    ]
+  },
+  {
+    path: 'login-landing',
+    component: LoginLandingRedesignedComponent,
+    canActivate: [IrbGuard]
+  },
```

Comment:
Please, add `login-landing/:mode` route like here - https://github.com/broadinstitute/ddp-angular/blob/develop/ddp-workspace/projects/ddp-angio/src/app/app-routing.module.ts#L122
We use it at creating a new account in case a user wants to sign in with existing account:
![image](https://user-images.githubusercontent.com/30126907/80356300-f0c7b500-8881-11ea-9edb-946ca2b4c012.png)

Also, please add `password-reset-done` path https://github.com/broadinstitute/ddp-angular/blob/develop/ddp-workspace/projects/ddp-angio/src/app/app-routing.module.ts#L170
And `session-expired` path https://github.com/broadinstitute/ddp-angular/blob/develop/ddp-workspace/projects/ddp-angio/src/app/app-routing.module.ts#L175